--- packages/nest-2.10.0/pynest/nest/__init__.py	2016-05-26 11:57:01.145861990 +0000
+++ packages/nest-2.10.0/pynest/nest/__init__.py_patched	2016-05-26 11:56:52.166557892 +0000
@@ -50,7 +50,7 @@


 from . import pynestkernel as _kernel
-import lib.hl_api_helper as hl_api
+from .lib import hl_api_helper as hl_api

 engine = _kernel.NESTEngine()

@@ -174,7 +174,7 @@
     hl_api.set_debug(debug)

 from .pynestkernel import *
-from lib.hl_api_helper import *
+from .lib.hl_api_helper import *

 # We search through the subdirectory "lib" of the "nest" module
 # directory and import the content of all Python files therein into
@@ -182,7 +182,7 @@
 # and those of extra modules available to the user.
 for name in os.listdir(os.path.join(os.path.dirname(__file__), "lib")):
     if name.endswith(".py") and not name.startswith('__'):
-        exec("from lib.{0} import *".format(name[:-3]))
+        exec("from .lib.{0} import *".format(name[:-3]))

 if not 'DELAY_PYNEST_INIT' in os.environ:
     init(sys.argv)
